src/actions/signupActions.js

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import { SIGNUP_ERROR, SIGNUP_SUCCESS, SIGNUP_STARTED } from '../actionTypes/signupTypes';
+
+export /**
```

Comment:
Why this? Why `export` is coming next to the jsdoc? It should be a problem with your `prettier` configuration. If it is possible, you can review your settings